src/app/interfaces/dialog.interface.ts

```diff
@@ -14,7 +14,7 @@ export interface ConfirmOptions {
 }
 
 export interface ConfirmOptionsWithSecondaryCheckbox extends ConfirmOptions {
-  secondaryCheckBox: true;
+  secondaryCheckBox: boolean;
```

Comment:
This was not a mistake. This was to specifically indicate that this interface and its fields kick in when secondaryCheckBox is true.